libtenzir/builtins/functions/time.cpp

```diff
@@ -126,74 +126,42 @@ class since_epoch final : public function_plugin {
   }
 };
 
-class from_epoch_ms final : public function_plugin {
+class from_epoch final : public function_plugin {
```

Comment:
Maybe deprecate instead of directly remove?

Reply:
It's a simple upgrade path, still so?

Reply:
I suspect that this isn't used widely, so I'd be fine with just removing it.